public/index.html

```diff
@@ -224,25 +237,59 @@ <h2 class="sr-only">Apply for Health Care</h2>
         </div>
       </div>
 
-      <div class="small-12 medium-8 columns text-right">
-        <a href="/2015/11/11/why-we-are-designing-in-beta.html" class="usa-button usa-button-outline-inverse">Help Us Create vets.gov: A Message from Secretary Robert McDonald</a>
-      </div>
-
       </div>
 
   </div>
 
-  <div data-alert class="va-notice--banner va-notice--banner--alt">
+  <div class="va-notice--banner va-notice--banner--alt">
     <div class="row">
       <div class="small-12 columns">
         <a id="feedback-tool" class="feedback-widget" tabindex="0" href="http://veterans.uservoice.com" data-uv-trigger="smartvote">
           <span class="icon"></span>
           <span class="text">Provide feedback on vets.gov</span>
         </a>
+        
+        <a href="https://veterans.uservoice.com/" class="feedback-widget">Visit the feedback forum</a>
       </div>
     </div>
   </div>
-</div>
+
+<div class="row va-util-rel"> <!-- VA Panel no touch screens -->
+  <div class="va-crisis-panel va-crisis-panel--notouch">
+    <button class="va-crisis-line va-crisis-line--notouch">Get help from Veterans Crisis Line</button>
+    <div  class="va-crisis-panel-body">
+      <ul>
+        <li>Call <strong>1-800-273-8255 (Press 1)</strong></li>
+        <li>Text to <b>838255</b></li>
+        <li><a href="https://www.veteranscrisisline.net/ChatTermsOfService.aspx?account=Veterans%20Chat">Chat confidentially now</a></li>
+      </ul>
+      <p>If you are in crisis or having thoughts of suicide, 
+visit <a href="https://www.veteranscrisisline.net/">VeteransCrisisLine.net</a> for more resources.</p>
+    </div>      
+  </div> 
+ <!--END VA Panel no touch screens -->   </div>
+</footer>
+
+
+<script>
+
+UserVoice=window.UserVoice||[];(function(){var uv=document.createElement('script');uv.type='text/javascript';uv.async=true;uv.src='//widget.uservoice.com/NDkEKHU7ZuU64RQKkUL9Q.js';var s=document.getElementsByTagName('script')[0];s.parentNode.insertBefore(uv,s)})();
+
+UserVoice.push(['identify', {
+}]);
+
+UserVoice.push(['autoprompt', {}]);
+</script>
+
+<!-- Google Tag Manager -->
```

Comment:
We already have a Google Tag Manager tag earlier in the file, so I don't think we need this one.